src/main/java/hu/oe/nik/szfmv/automatedcar/powertrain/ITransmission.java

```diff
@@ -0,0 +1,29 @@
+package hu.oe.nik.szfmv.automatedcar.powertrain;
+
+/**
+ * ITransmission interface
+ * Describes a transmission class.
+ */
+public interface ITransmission {
+
+    /**
+     * Getter method for gearMode field.
```

Comment:
As getter documentation, we rather describe the value given back, not its implementation details. The one reading the JavaDoc of the getter is interested in what value it gives in what circumstances, not which private field it refers to.
Just a side note, otherwise I like the effort on the JavaDocs :)